share/dkg/pedersen/dkg.go

```diff
@@ -343,6 +380,32 @@ func (d *DistKeyGenerator) DistKeyShare() (*DistKeyShare, error) {
 	}, nil
 }
 
+
+func (f *DistKeyShare) Renew(g *DistKeyShare, suite Suite) (*DistKeyShare, error){
```

Comment:
Please put the suite as the first argument. It seems an annoying request but it's much much easier when the code is consistent with the rest of the repo. :)

Reply:
Also you need to write some comments on this function explaining from a higher level point of view what it is doing.